From 1fe90d3f2ab64bd98e3e659190e541cfc3f77457 Mon Sep 17 00:00:00 2001
From: Yang Shen <shenyang39@huawei.com>
Date: Fri, 3 Dec 2021 18:16:05 +0800
Subject: [PATCH] config: enable SVA feature
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 1fe90d3f2ab64bd98e3e659190e541cfc3f77457


driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4J9WR

----------------------------------------------------------------------

Enable the config about SVA feature.

Signed-off-by: Yang Shen <shenyang39@huawei.com>
Reviewed-by: Hao Fang <fanghao11@huawei.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index d94a72fd0ffe..73cd7d9c9a78 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -5812,11 +5812,12 @@ CONFIG_IOMMU_IO_PGTABLE_LPAE=y
 # CONFIG_IOMMU_DEFAULT_PASSTHROUGH is not set
 CONFIG_OF_IOMMU=y
 CONFIG_IOMMU_DMA=y
+CONFIG_IOMMU_SVA_LIB=y
 CONFIG_ARM_SMMU=y
 # CONFIG_ARM_SMMU_LEGACY_DT_BINDINGS is not set
 CONFIG_ARM_SMMU_DISABLE_BYPASS_BY_DEFAULT=y
 CONFIG_ARM_SMMU_V3=y
-# CONFIG_ARM_SMMU_V3_SVA is not set
+CONFIG_ARM_SMMU_V3_SVA=y
 CONFIG_SMMU_BYPASS_DEV=y
 # CONFIG_QCOM_IOMMU is not set
 # CONFIG_VIRTIO_IOMMU is not set
-- 
2.26.2

